Starcounter.Uniform/ViewModels/UniDataTable.json.cs

```diff
@@ -109,11 +125,16 @@ public void Handle(Input.Filter action)
                     });
                 }
 
-                LoadRows?.Invoke();
+                LoadRows?.Invoke(true);
             }
 
             public void Handle(Input.Sort action)
             {
+                if (action.Value != "asc" && action.Value != "desc" && !string.IsNullOrEmpty(action.Value))
```

Comment:
Instead of hardcoding "asc" and "desc" here, refer to a definition used by parsing code